uillinois-community/uillinois-community.github.io
# WIP: Bare minimum UIllinois branding

## Summary

Adds bare minimum layout rules and UIllinois branding to [community page](https://uillinois-community.github.io). Uses precompiled bootstrap, which is overkill, but was quick to implement.

## Testing

0. clone project, switch to branding branch, and open index.html in browser. (UIllinois Community does not have officially supported browser list.)
0. Check that content is readable at all screen sizes
0. Check that logo placement meets [uillinois brand requirements](https://www.uillinois.edu/OUR/brand/system_logos).
0. Check that type rules meet [uillinois brand requirements](https://www.uillinois.edu/OUR/brand/style)
0. Check that color use meets [uillinois brand requirements](https://www.uillinois.edu/OUR/brand/color_palettes)

## Future Enhancements

- Consider reducing css weight
- Consider aligning color palette more closely with UIllinois brand standards
